src/dev/src/app/radios/radios.demo.html

```diff
@@ -15,14 +15,19 @@ <h4>Vertical</h4>
       <div class="clr-radio-wrapper">
         <input type="radio" id="vertical-radio1" name="radio-basic" class="clr-radio">
         <label for="vertical-radio1" class="clr-control-label">option 1</label>
+      </div>
+      <div class="clr-radio-wrapper">
         <input type="radio" id="vertical-radio2" name="radio-basic" class="clr-radio">
         <label for="vertical-radio2" class="clr-control-label">option 2</label>
+      </div>
+      <div class="clr-radio-wrapper">
         <input type="radio" id="vertical-radio3" name="radio-basic" class="clr-radio">
         <label for="vertical-radio3" class="clr-control-label">option 3</label>
       </div>
-      <!-- IMPORTANT DIFFERENCE IN STRUCTURE! ICON IS NOT PART OF THE INPUT WRAPPER -->
-      <clr-icon class="clr-validate-icon" shape="exclamation-circle"></clr-icon>
-      <span class="clr-subtext">Helper Text</span>
+      <div class="clr-subtext-wrapper">
```

Comment:
@mathisscott This is actually what is different, but not documented yet. It works without this wrapper unless you have inline, so it should probably just be the default option regardless.